examples/blog-update/overrides/partials/post.html

```diff
@@ -0,0 +1,67 @@
+{#-
```

Comment:
The banner should be removed, since this file was not automatically generated but manually overridden 😉

Reply:
I have used the space for a note about updates. Hope that makes sense. It is now mentioned in the README as well.